packages/oui-field/src/field.component.js

```diff
@@ -7,7 +7,8 @@ export default {
         helpText: "@?",
         size: "@?",
         errorMessages: "<?",
-        labelPopover: "@?"
+        labelPopover: "@?",
+        errorDisplay: "&?"
```

Comment:
You don't need to specify the optional option :
```suggestion
        errorDisplay: "&"
```

Reply:
A question here could be: Do we need a callback in this case, instead of just a boolean attribute ? 

Currently, the errors are displayed when the field is `$dirty && $invalid` and blurred. But the error disappears when the field gets back the focus (eg. https://ovh-ux.github.io/ovh-ui-kit/?path=/story/components-field--input).

Maybe we could just simplify this part by removing the "focus" part. To be discussed :)